content-repo/extra-docs/releases/20.6.1.md

```diff
@@ -0,0 +1,599 @@
+
+---
+
+### Plain Text Feed Pack v1.0.1
+#### Integrations
+##### Plain Text Feed  
+Changed the docker image to support auto-detection function.
+
+---
+
+### Prisma Access Pack v1.0.1
+#### Playbooks
+##### Prisma Access Whitelist Egress IPs on SaaS Services
+- Added a call for Okta Zones subplaybook.
+- Moved the names of the AWS security group, GCP firewall and Okta Zone into Playbook inputs. If the input is not set - the related subplaybook will be skipped.
+
+---
+
+### Prisma Cloud Pack v1.2.0
+#### Integrations
+##### RedLock  
+- Added defAdded default classifier and mapper.ault classifier and mapper.
+- Added support for multi-environment instances.
+- Added the ***get-remediation-details*** command.
+
+#### Playbooks
+##### Prisma Cloud Remediation - GCP VPC Network Misconfiguration
+- New Prisma Cloud remediations:
+  - GCP Default Firewall rule should not have any rules (except http and https)
+  - GCP Firewall with Inbound rule overly permissive to All Traffic
+##### Prisma Cloud Remediation - GCP VPC Network Firewall Misconfiguration
+- New Prisma Cloud remediations:
+  - GCP Default Firewall rule should not have any rules (except http and https)
+  - GCP Firewall with Inbound rule overly permissive to All Traffic
+
+---
+
+### Proofpoint Protection Server Pack v1.0.2
+#### Integrations
+##### Proofpoint Server Protection
+- Improved parsing of responses returned from Proofpoint.
+- Added support for Proofpoint Protection Server version 8.14.2.
+
+---
+
+### RTIR Pack v1.0.2
+#### Integrations
+##### RTIR  
+- Fixed an issue where headers with 'ID' in their name got malformed when running ***rtir-search-ticket*** command and in ***fetch-incidents***.
+- Improved ticket attachment parsing
+
+---
+
+### Recorded Future Feed Pack v1.0.1
+#### Integrations
+##### Recorded Future Feed  
+Improved parsing of IOC objects.
+
+---
+
+### Red Canary Pack v1.0.2
+#### Integrations
+##### RedCanary
+- Removed timeline details for a detection fetched as an incident.
+- Fixed an issue in which acknowledged detections were fetched as incidents.
+- Improved processing of outputs for endpoint details.
+
+---
+
+### Securonix Pack v1.1.1
+#### Integrations
+##### Securonix
+- Added max parameter to the `securonix-list-incidents` command
+- Added the `max_fetch` parameter to the integration configuration, where the default and maximum value is 50.
+- Fixed an issue where duplicate incidents where fetched.
+
+---
+
+### SentinelOne Pack v1.0.1
+#### Integrations
+##### SentinelOne V2  
+Improved processing of datetime strings.
+
+---
+
+### ServiceNow Pack v1.1.4
+#### Integrations
+##### ServiceNow v2  
+Improved handling of authentication errors returned from ServiceNow.
+
+---
+
+### Slack Pack v1.2.1
+#### Integrations
+##### SlackV2  
+Added stability improvements for long-running execution.
+
+---
+
+### TIM - Processing Pack v1.1.1
+#### Playbooks
+##### TIM - Indicator Auto Processing  
+Added sub playbook for Whois.
```

Comment:
Was released already last time. Can be removed. @yaakovi looks like we have a bug in the script.